examples/from_source/WORKSPACE

```diff
@@ -0,0 +1,57 @@
+workspace(
+    name = "from_source",
+    managed_directories = {"@npm": ["node_modules"]},
+)
+
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+
+git_repository(
+    name = "build_bazel_rules_nodejs",
+    commit = "981e7c1ab431c79dd83bcf5d495fefc4a0e02e45",
+    remote = "https://github.com/bazelbuild/rules_nodejs.git",
+    shallow_since = "1595023729 -0700",
+)
+
+git_repository(
+    name = "build_bazel_rules_typescript",
+    commit = "10a5a86885f95ab788fd841ade47b6a16e0c13d6",
+    patches = [
+        "@build_bazel_rules_nodejs//:rules_typescript_pr_494.patch",
```

Comment:
we should move this into a reusable function - I added another patch in a PR I put up yesterday

Reply:
Added to package.bzl